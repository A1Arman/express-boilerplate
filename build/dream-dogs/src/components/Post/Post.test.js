import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Post from "./Post";

describe(`Post component`, () => {
  it("renders the complete form", () => {
    const wrapper = shallow(<Post />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});