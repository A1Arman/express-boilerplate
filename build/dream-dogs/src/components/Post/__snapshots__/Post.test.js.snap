// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post component renders the complete form 1`] = `
<main
  className="flex-container"
>
  <header>
    <h1
      className="post-title"
    >
      New Post
    </h1>
  </header>
  <section
    className="post-form-section"
  >
    <form
      className="post-form"
      id="post-form"
    >
      <div
        className="form-section"
      >
        <label
          htmlFor="dog_name"
        >
          Dog Name
        </label>
        <input
          name="dog_name"
          placeholder="Jax"
          required={true}
          type="text"
        />
      </div>
      <div
        className="form-section"
      >
        <label
          htmlFor="breed"
        >
          Breed
        </label>
        <input
          name="breed"
          placeholder="Maltipoo"
          required={true}
          type="text"
        />
      </div>
      <div
        className="form-section"
      >
        <label
          htmlFor="dog_bday"
        >
          Birthdate
        </label>
        <input
          name="dog_bday"
          type="date"
        />
      </div>
      <div
        className="form-section"
      >
        <label
          htmlFor="lifestyle"
        >
          Dog Lifestyle
        </label>
        <textarea
          name="lifestyle"
          placeholder="Please enter a description of your dogs needs and lifestyle"
          required={true}
          rows="15"
        />
      </div>
      <button
        className="sub-btn"
        type="submit"
      >
        Submit
      </button>
      <button
        className="res-btn"
        type="reset"
      >
        Reset
      </button>
    </form>
  </section>
</main>
`;
